### DISCUSSION TITLE
Coin on GLES for supporting Vulkan with Angle

### DISCUSSION THREAD

**Discussion by famorcia** [coin3d/coin#494] [Ideas]
I would propose moving Coin on GLES.
This could extend usage on more platforms (e.g. Android and IOS) and move on Vulkan through the [Angle](https://github.com/google/angle) library.

**Comment by WizzerWorks** [comment]
I would also like to see Coin on OpenGL ES, but for use on Linux embedded platforms.

**Comment by WizzerWorks** [comment]
I created a private project, OpenGL ES Port, to coordinate efforts for supporting GLES. Please let me know if you are interested and I will add you to the project.

**Reply by famorcia** [reply]
Hello, 
I'm not really expert on GLES and embedded platform (yet), but I hope to provide some help.

**Reply by SummerTree** [reply]
Hi, is any progress on this project? I want to use OpenGL ES version Coin3d,too. Could you add me to your project? Thank you very much.

**Reply by famorcia** [reply]
Hello, sorry for the late answer. I'm not working in this topic since last message. The coin3d branch seems to be stopped at that time.

**Comment by famorcia** [comment]
Hello Mark,
some feedback about (Investigate previous GLES efforts) on:
[epuzanovv EGL](https://github.com/epuzanov/coin/tree/glue_egl)
[famorcia EGL](https://github.com/famorcia/coin_egl): Fork of epuzanovy

epuzanovv EGL is not aligned to the last master. Some test on ubuntu shows that the finder uses both GLX and EGL and this is a problem on final executable. EGL should be managed as dynamic loading or as alternative to GLX.

famorcia EGL is mainly related to adding Mesa Offscreen Rendering, does not add special task on EGL (yet)
EGL could be a good replace for pbuffer and offscreen rendering that now is not working on coin.

Cheers

   Fabrizio
